scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Those variables, since it's user input, should all be in the cell above where EPOCHS is specified, and in capital letters. Likely though we only want to highlight the 2-3 important parameters. All other parameters we'll set defaults in the utils_cv implementation and won't over-write (or even mention) here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>